core/sts-3.3.md

```diff
@@ -23,11 +23,15 @@ MUST only connect to the server securely (using TLS, aka. SSL), and the port
 number to use for it.
 
 Clients MUST use sufficient certificate verification to ensure the connection
-is secure without any errors.
+is secure without any errors. This may involve validation against a system root
+certificate bundle or a user-specified trust root.
+
+In compliant server software, this capability MUST be made available as an optional
+configuration setting. Server administrators may have valid reasons not to enable it.
 
 ## Details
 
-The capability value, which MUST always be present, is a comma (`,`) separated
+The capability value, which MUST always be present when enabled, is a comma (`,`) separated
```

Comment:
I think you mean the capability should be present when enabled, not only its value.